src/scss/layout/_video.scss

```diff
@@ -0,0 +1,57 @@
+@mixin  fontStyle_video ($color, $fontSize, $fontWeight) {
+    color: $color;
+    font-size: $fontSize;
+    font-weight: $fontWeight;
+    line-height: 28px;
+    font-family: Helvetica;
+    font-style: normal;
+}
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0px;
+}
+
+body {
+    max-width: 1440px;
+    margin: auto;
+}
```

Comment:
remove these styles in the body, and use the class .component which will implement the layout.

Reply:
Done.

Reply:
it is still pending